lib/beaker-hostgenerator/data.rb

```diff
@@ -1082,6 +1082,11 @@ def generate_osinfo
         yield ["oracle#{release}-64", "el-#{release}-x86_64"]
       end
 
+      # OracleLinux
```

Comment:
How is this different from oracle just above it?

Reply:
I was already wondering where that's coming from. We currently don't use that anywhere. Is that maybe a leftover that we can delete? Is Puppet using that internally for oracle builds?

Reply:
That's my assumption as well. We always use `OracleLinux` in metadata.json so I think we should use that here as well. I can raise another PR to remove the old code and we merge that when we do another major release.

Reply:
As I said in https://github.com/voxpupuli/beaker-hostgenerator/pull/322#pullrequestreview-1567195970, I'd like to continue using `oracle` as the name.

It avoids doing a major release (which is painful) and in daily operation it's also easier.